extension/chrome/settings/modules/change_passphrase.ts

```diff
@@ -74,6 +73,7 @@ View.run(class ChangePassPhraseView extends View {
       await Ui.modal.error('Pass phrase did not match, please try again.');
       $('#current_pass_phrase').val('').focus();
     }
+    Assert.abortAndRenderErrorIfKeyinfoEmpty(this.primaryKi);
```

Comment:
This one may have been added by mistake

Reply:
@tomholub Yeah, It was added by mistake while resolving the merge conflicts